### PR TITLE
Some refinements and additions

### PR DESCRIPTION
**Pull request by tomayac**
- Consistency fixes, grammar, commas…
- Some missing links.
- Suggested the `:nth-of-type()` pseudo class.
- Suggested the [case sensitivity](https://www.w3.org/TR/selectors-4/#attribute-case) attribute.

### CODE REVIEW

File: EXTENSIONS.md [hunk 1]
```diff
@@ -221,55 +231,56 @@ first before we could know whether it matches.
 
 ### Why use the WebAnnotations syntax?
 
-There's a few advantages to reusing the already existing syntax offered by
+There're a few advantages to reusing the already existing syntax offered by
```
Comment (bokand) [inline EXTENSIONS.md:234]:
```suggestion
There are a few advantages to reusing the already existing syntax offered by
```

**Review by bokand — APPROVED** [review]
Thanks @tomayac! LGTM

File: EXTENSIONS.md (hunk 1 above)
Comment (tomayac) [reply EXTENSIONS.md]: Fixed throughout.

**Comment by tomayac** [comment]
@bokand PTAL, I have added an image ([preview](https://github.com/WICG/scroll-to-text-fragment/blob/67e1e145637c13fdca261669b5e9866b2c1471a3/EXTENSIONS.md)) to illustrate the basic idea.

**Comment by bokand** [comment]
Nice, thanks! still lgtm

**Review by bokand — APPROVED** [review]

**Pull request merged**